### PR TITLE
argparse stub fix: parse_known_args takes an optional sequence of strings like parse_args(), not an optional string

### PR DESCRIPTION
**Pull request by jdelic**
The current definition of `parse_known_args` seems simply wrong.

### CODE REVIEW

**Comment by gvanrossum** [comment]
Can you rebase?

**Comment by jdelic** [comment]
sure, on it

**Comment by jdelic** [comment]
ok... I seemed to have done a stupid by trying to be too clever with Git, but now the rebase seems to have worked. All ready, once the CI is done.

File: stdlib/2and3/argparse.pyi [hunk 1]
```diff
@@ -79,8 +79,9 @@ class ArgumentParser:
     def print_help(self, file: Optional[IO[str]] = ...) -> None: ...
     def format_usage(self) -> str: ...
     def format_help(self) -> str: ...
-    def parse_known_args(self, args: Optional[str] = ...,
-                         namespace: Optional[Namespace] = ...) -> Tuple[Namespace, List[str]]: ...
+    def parse_known_args(self, args: Optional[Sequence[str]] = ...,
+                         namespace: Optional[Namespace] = ...) \
```
Comment (gvanrossum) [inline stdlib/2and3/argparse.pyi]: Can you avoid using \ for line continuations? Someone else is doing this (apparently the pytype parser doesn't like them yet).

**Comment by jdelic** [comment]
sure, absolutely. I think my IDE might have been responsible for this. I will avoid them for PRs in the future.

**Pull request merged**